src/client/errorMiddleware.js

```diff
@@ -3,15 +3,16 @@ import errors from '../common/constants/errors';
 import { notify } from './app/Notification/notificationActions';
 
 function parseBlockChainError(error) {
-  const errorType = _.find(errors, e => _.includes(error, e.fingerprint));
+  const errorType = _.find(errors, e => error.includes(e.fingerprint));
 
   if (_.has(errorType, 'message')) {
     return errorType.message;
   }
-
-  // Log error to console for further investigation.
-  console.log('Unknown error', error);
-  return 'Unkown error has occured.';
+  const idx = error.indexOf(':');
+  if (idx > 0) {
+    return error.slice(idx + 1).trim();
+  }
+  return error;
```

Comment:
I think it should return `Unknown error` there.

Reply:
Yeah you can use "Unknown error" and console log the error so we could debug it for user.